Backend/app/core/tts.py

```
@@ -0,0 +1,307 @@
+            self.engine = pyttsx3.init()
+            voices = self.engine.getProperty('voices')
+            if voices:
+                logger.info(f"📋 Available TTS voices: {len(voices)}")
+                for i, voice in enumerate(voices[:3]):
+                    logger.info(f"   Voice {i}: {voice.name if hasattr(voice, 'name') else 'Unknown'}")
+            else:
+                logger.warning("⚠️ No TTS voices found on system")
+            logger.info("✅ pyttsx3 TTS engine initialized")
+        except Exception as e:
+            logger.error(f"❌ Failed to initialize TTS engine: {e}")
+            raise Exception(f"TTS engine initialization failed: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)